mcs/packages/Makefile

```diff
@@ -3,7 +3,7 @@ include ../build/rules.make
 
 ROSLYN_CSC_DIR = $(dir $(CSC_LOCATION))
 
-ROSLYN_FILES = \
+ROSLYN_FILES_FOR_MONO = \
```

Comment:
Did you mean `lib/mono/msbuild/15.0/bin/Roslyn` -> `lib/mono/4.5/` ? We could do that too. Though having an actual directory, let's us have more control over the specific `Roslyn` folder meant for `msbuild` - allowing us to add/remove files, if necessary (based on what upstream msbuild does!)